custom_components/ojmicroline_thermostat/sensor.py

```diff
@@ -79,14 +83,6 @@ def _temp_formatter(temp: Any) -> float:
     return temp / 100
 
 
-def _date_formatter(dt: Any) -> datetime | None:
-    """Format the date."""
-    now = datetime.now(dt.tzinfo)
-    if now > dt:
```

Comment:
This logic is now gone; we always return dates if they are relevant to the current mode, even if they're in the past.